x-pack/plugins/observability/public/pages/slo_edit/index.test.tsx

```diff
@@ -163,19 +165,61 @@ describe('SLO Edit Page', () => {
       );
     });
 
-    it('renders the SLO Edit page with prefilled form values if sloId route param is passed', async () => {
+    it.skip('calls the createSlo hook if all required values are filled in', async () => {
```

Comment:
💬 Disabled for now as it takes 4s to wait for the submit button to be enabled... And I can't figure out why.